### ISSUE TITLE
[A11Y] [Bug]: Links should be underlined

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#757]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14 Plus

### OS Version

17.1.1

### App version

v0.18.0

### Describe the issue

Links are not clearly displayed except from the colour
![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/0b050575-5cdf-4274-b4c2-d5e5202b5cae)

### Expected Behavior

add .underline() to make links visually distinct
